src/main/webapp/app/home/home.component.html

```diff
@@ -37,26 +37,35 @@ <h1 jhiTranslate="home.title"></h1>
                                 class="form-control"
                                 id="username"
                                 name="username"
-                                placeholder="{{ 'global.form.username.placeholder' | artemisTranslate }}"
+                                [placeholder]="usernamePlaceholderTranslated"
```

Comment:
Why did you change this?
Is this approach better than the pipe?

Reply:
please double check the component code, we have different cases (based on the profile information from the server and need to be able to adapt this)